src/ui/hexeditwidget.cc

```diff
@@ -218,22 +218,25 @@ void HexEditWidget::createActions() {
   connect(auto_resize_checkbox_, &QCheckBox::toggled,
           [this](bool toggled) { hex_edit_->setAutoBytesPerRow(toggled); });
 
-  change_edit_mode_button_ = new QPushButton("override");
+  const QString& overwrite_label_ = QStringLiteral("Overwrite");
+
+  change_edit_mode_button_ = new QPushButton(overwrite_label_);
   connect(change_edit_mode_button_, &QPushButton::clicked,
           [this]() { change_edit_mode_act_->trigger(); });
 
   change_edit_mode_act_ = ShortcutsModel::getShortcutsModel()->createQAction(
       util::settings::shortcuts::CHANGE_EDIT_MODE, this,
       Qt::WidgetWithChildrenShortcut);
-  connect(change_edit_mode_act_, &QAction::triggered, [this]() {
-    if (hex_edit_->isInsertMode()) {
-      hex_edit_->setInsertMode(false);
-      change_edit_mode_button_->setText("override");
-    } else {
-      hex_edit_->setInsertMode(true);
-      change_edit_mode_button_->setText("insert");
-    }
-  });
+  connect(change_edit_mode_act_, &QAction::triggered,
+          [this, overwrite_label_]() {
```

Comment:
Should be `&overwrite_label_`. A reference captured by value causes copying of the original object. We want a reference to `QStringLiteral("Overwrite")` (which is a static object, so this is ok).